packages/components/pie-cookie-banner/test/helpers/page-object/selectors.ts

```diff
@@ -0,0 +1,51 @@
+const cookieBanner = {
+    selectors: {
+        container: {
+            description: 'The selector for the cookie banner container',
+            dataTestId: 'pie-cookie-banner',
```

Comment:
Would it be a bad idea to import these into the component?

Reply:
Yeah, so ideally, we'd be going against the page object model by importing these into our component directly as generally, consumers will only be using the functionality we'd provide through our page object. By exposing these in the tests, we'd essentially be giving them free reign to start exploring the rest of the component when the main focus for consumers is to just use our testing structure.